msm/__main__.py

```diff
@@ -54,8 +54,6 @@ def main(args=None, printer=print):
                         default='default')
     parser.add_argument('-u', '--repo-url')
     parser.add_argument('-b', '--repo-branch')
-    parser.add_argument('-d', '--skills-dir')
-    parser.add_argument('-c', '--repo-cache')
```

Comment:
i would like to keep skills-dir as an option, msm itself is not part of core but a command line tool, this just reduces how useful it is

what if i want to inspect a skill before essentially giving it root access? mycroft security model is quite bad and a malicious skill is essentially game over, and since the name matching is less than ideal its also easy to install the wrong thing. 

by setting skills_dir it allows to inspect a skill before it is run. you can argue we should just git clone manually for this, but kinda defeats the purpose of free text search, and hides the commit pinned by the marketplace

Reply:
Yeah I wondered if there'd be a use case like this.

It shouldn't be too hard to retain support for these though right?

It's completely up to the person using these flags to ensure the directories have the correct permissions etc.

Reply:
Oh well, it makes the tests easier as well. Re-added it.